src/lib/installation/clients/inst_download_release_notes.rb

```diff
@@ -122,5 +131,28 @@ def refresh_ui(relnotes_map)
         Wizard.ShowReleaseNotesButton(_("Re&lease Notes..."), "rel_notes")
       end
     end
+
+    # Fetch the release notes for a given product
+    #
+    # @param product [Y2Packager::Product]
+    # @param user_lang [String] Preferred language (use current language as default)
+    # @param format    [Symbol] Release notes format (use :txt as default)
+    # @return [ReleaseNotes,Symbol] Release notes for product, language and format.
+    #   :missing if release notes are not present and :error if something went wrong
+    # @see Y2Packager::Product#release_notes
+    def fetch_release_notes(product, user_lang, format)
+      product.release_notes(user_lang, format) || :missing
+    rescue Y2Packager::PackageFetchError, Y2Packager::PackageExtractionError => e
+      log.warn "Could not download and extract the release notes package for '#{product.name}': #{e.inspect}"
+      :error
+    end
+
+    # Displays
+    def display_warning(products)
+      # TRANSLATORS: 'product' stands for the product's name
+      msg = HTML.Para(_("An error occurred when retrieving the release notes for the following products:")) +
```

Comment:
The screenshot says "Warning", "An error occurred...". That is a bit confusing, so is it an error or just a warning? :thinking: 

Maybe just a bit rewording the message would help, like

```suggestion
      msg = HTML.Para(_("The release notes for the following products could not be retrieved:")) +
```

Do not forget to adapt the tests as well.